tests/test_mv3_downloads_contract.py

```
@@ -0,0 +1,138 @@
+
+        error = runner.CompatibilitySurfaceError(
+            {
+                "downloads": "missing",
+                "downloadsDiagnostic": "download-source-rejected",
+            }
+        )
+        evidence = runner._failure_evidence(error)
+        self.assertEqual(
+            evidence["observed"]["downloadsDiagnostic"], "download-source-rejected"
+        )
+        self.assertNotIn("/tmp/private", repr(evidence))
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/private"

(S108)